src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -3323,6 +3327,10 @@ private BoundStatement BindReturn(ReturnStatementSyntax syntax, DiagnosticBag di
                     {
                         Error(diagnostics, ErrorCode.ERR_RefReturnMustHaveIdentityConversion, argument.Syntax, returnType);
                     }
+                    else
+                    {
+                        return argument;
+                    }
```

Comment:
Same as above - there is no conversion that takes place here. 
Which way would conversion go? From the arg type to the return type or the other way?

---
In reply to: [81204314](https://github.com/dotnet/roslyn/pull/14176#discussion_r81204314) [](ancestors = 81204314)